packages/console/src/pages/ApplicationDetails/index.tsx

```diff
@@ -144,13 +144,16 @@ const ApplicationDetails = () => {
               </div>
             </div>
             <div className={styles.operations}>
-              <Button
-                title="application_details.check_guide"
-                size="large"
-                onClick={() => {
-                  setIsReadmeOpen(true);
-                }}
-              />
+              {/* TODO: @Charles figure out a better way to check guide availability */}
```

Comment:
nit: I think we should have had guide for all application types. So IMO providing a guide for M2M should be good enough?

Linear task added to create M2M guide
https://linear.app/silverhand/issue/LOG-4228/providing-integration-guide-for-m2m-application

Reply:
yes, that's in the to-do list. thanks